src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -36,6 +36,7 @@
         "symfony/phpunit-bridge": "~2.7|~3.0.0",
         "symfony/browser-kit": "~2.4|~3.0.0",
         "symfony/console": "~2.6|~3.0.0",
+        "symfony/config": "~2.7|~3.0.0",
```

Comment:
@fabpot is it right to add these? Otherwise a too low version of symfony/config is installed.